### PR TITLE
[IMP] finance: remove fiscal localization countries screenshots

### PR DESCRIPTION
**Pull request by StraubCreative**
**Overview**
Remove confusing screenshots for customers under **Finance > Accounting and Invoicing > Fiscal Localizations** section. Customers get confused by the screenshots of the country modules, as they think that those modules are the _only_ country-supported modules we have (which makes more issues than not).

**Summary of changes**
- Remove country modules screenshots across `localizations_list.rst` and `fiscal_localization_packages.rst`
- Delete _/media_ folder in _/applications/finance/accounting/fiscal_localizations/overview_ which contain `fiscal_localization_packages_selection.png` and `fiscal_localization_packages_modules.png`

**Outcomes**
- Removed images will hopefully remove confusion for the customer and will be more likely to follow the text directions which encourage them to look through the entire modules list. 
- Small page-speed improvement with lighter page weight.

Requested by: @jubodoo

### CODE REVIEW

File: content/applications/finance/accounting/fiscal_localizations/overview/fiscal_localization_packages.rst [hunk 1]
```diff
@@ -22,10 +22,6 @@ selected at the creation of the database.
 To install a new package, go to :menuselection:`Accounting --> Configuration --> Fiscal
 Localization`, click on **Install More Packages**, and install your country's module.
 
-.. image:: media/fiscal_localization_packages_modules.png
-   :align: center
-   :alt: Install the appropriate module as fiscal localization package in Odoo Accounting.
-
 Once done, select your country's package, and click on *Save*.
 
 .. image:: media/fiscal_localization_packages_selection.png
```
Comment (AntoineVDV) [inline content/applications/finance/accounting/fiscal_localizations/overview/fiscal_localization_packages.rst:27]: You deleted the image but did not remove references to that image, hence the red runbot build.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: content/applications/finance/accounting/fiscal_localizations/overview/fiscal_localization_packages.rst (hunk 1 above)
Comment (StraubCreative) [reply content/applications/finance/accounting/fiscal_localizations/overview/fiscal_localization_packages.rst:27]: Ah! Silly mistake. Thanks for catching.
Should be resolved on latest commit [931710b](https://github.com/odoo/documentation/pull/1311/commits/931710b52b063e3d6a853713dbdb8d766d406f76). Runbot built passed.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
LGTM but the commits need to be squashed into a single commit. Then we're good to go :)

**Comment by StraubCreative** [comment]
> LGTM but the commits need to be squashed into a single commit. Then we're good to go :)

Squashed in [0df4d3b](https://github.com/odoo/documentation/pull/1311/commits/0df4d3b80e23e41df3384b2b4a11ad1fb82176e9).

Good to go?

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**